packages/cli/src/commands/generate/helpers.ts

```diff
@@ -78,7 +93,7 @@ export const createYargsForComponentGeneration = ({
             },
           },
         ],
-        { collapse: false, exitOnError: true }
```

Comment:
Listr types claim `collapse` parameter doesn't exist?

Reply:
This seems like an oversight on my part. I swear this did something! but I can't seem to find any references to this in the documentation. The only thing I can imagine is that this is passed along to a custom renderer... but that just seems strange, and we're not using a custom renderer.

Reply:
It comes from this package which seems to be a dependency of Listr:
https://github.com/SamVerschueren/listr-update-renderer

And it _is_ definitely important for the task list Title display! (Although there may be another, better implementation now). When you have nested Tasks, you need this in order to keep **all tasks Titles displayed** once the array has run. Here's an example from my current work on CLI upgrade command:
<img width="585" alt="Screen Shot 2020-05-18 at 10 26 08 AM" src="https://user-images.githubusercontent.com/2951/82256268-dfb31680-990a-11ea-915e-322f41a8adf3.png">

^^ Without `collapse: false`, the indented titles (e.g. "Found @redwoojs/auth") would be hidden.

Reply:
Ah, you can see it mentioned here in the docs:

> Listr([tasks], [options])
> options
> Any renderer specific options. For instance, when using the update-renderer, you can pass in all of its options.

https://github.com/SamVerschueren/listr#options

So, I'm guessing that `listr-update-renderer` is the default renderer?

Reply:
Okay, I re-added the collapse option :+1: